subprojects/packagefiles/m4/src/nononnull.h

```diff
@@ -0,0 +1,3 @@
+#ifndef _GL_ARG_NONNULL
+#define _GL_ARG_NONNULL(params)
+#endif
```

Comment:
This should be a `.h.in` file that is configured with the copy only kwarg to stick with the requirement that we don't ship "active code" for lack of a better term.